explorer/nodewatch/openapi/openapi.yaml

```diff
@@ -0,0 +1,477 @@
+openapi: 3.0.3
+info:
+  title: NodeWatch API Documentation
+  description: |
+    API providing information about the NEM and Symbol blockchain networks, 
+    including node and network statistics and time-series.
+  version: 1.0.0
+
+servers:
+  - url: https://nodewatch.symbol.tools/testnet
+    description: Testnet server
+  - url: https://nodewatch.symbol.tools
+    description: Mainnet server
+
+tags:
+  - name: NEM nodes
+    description: Endpoints providing NEM node and network height information.
+  - name: Symbol nodes
+    description: Endpoints providing Symbol node and network height information, along with statistics.
+
+paths:
+  /api/nem/nodes:
+    get:
+      summary: Returns the list of known NEM nodes.
+      tags:
+        - NEM nodes
+      operationId: getNemNodes
+      responses:
+        200:
+          description: The list of known NEM nodes.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Node'
+
+  /api/nem/height:
+    get:
+      summary: Returns information about the NEM blockchain's heights.
+      tags:
+        - NEM nodes
+      operationId: getNemHeight
+      responses:
+        200:
+          description: The current and finalized heights.
```

Comment:
I just realized we need to explain what "current height" means, because each node in the network can have a different height.
We need to explain what is the height that Nodewatch returns. The maximum? minimum? average? a random node's height?
Same comment for Symbol.

Reply:
Added information. It's median from all values reported by node.